newsfragments/46944.significant.rst

```diff
@@ -0,0 +1,16 @@
+Statically-coded executors are now deprecated
+
+The Statically-coded executors ``LocalKubernetesExecutor`` and ``CeleryKubernetesExecutor`` are now deprecated in favor of Multiple Executor Configuration. They will be removed in Airflow 3.0
+
+* Types of change
+
+  * [x] Dag changes
+  * [x] Config changes
+  * [ ] API changes
+  * [ ] CLI changes
+  * [ ] Behaviour changes
+  * [ ] Plugin changes
+  * [ ] Dependency changes
+  * [ ] Code interface changes
+
+* Migration rules needed
```

Comment:
I am not super sure but I _think_ the classification is mainly for Airflow 3 - but as you are adding this to v2-10-test I think this is not needed.

Reply:
Yeah, I felt it might just be a helpful heads up. But you're right, it's probably more confusing than it is helpful. I'll trim that portion out :+1: